README.md

```diff
@@ -188,4 +188,4 @@ Note that once you've [submitted a pull request](https://github.com/Azure/platfo
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
\ No newline at end of file
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
```

Comment:
confused as to what changed down here but LGTM

Reply:
Not sure either, it seems maybe a line ending?